### PR TITLE
Staff can confirm student exam attendance.

### PR DESCRIPTION
**Pull request by spark008**

### CODE REVIEW

File: test/fixtures/exams.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 main_exam:
   assignment: main_exam
-  requires_confirmation: true
+  requires_confirmation: true  # should be true
```
Comment (pwnall) [inline test/fixtures/exams.yml]: The comment looks a bit silly. Do you mean something like "do not change, or many tests will fail"?

**Comment by pwnall** [comment]
LGTM with that small nit. Please rebase and feel free to merge after fixing the comment.

**Comment by spark008** [comment]
We have comments of that sort sprinkled throughout the fixtures. I thought it's better to be explicit about what the value should be, in case things accidentally get change. For times, it's also nice to be able to tweak the exact time slightly if you need to, and have a comment like 'should be in the past' so you know what range of times is acceptable for existing tests.

**Comment by pwnall** [comment]
My point was that `key: true   # must be true`, while more reassuring than `key: false  # must be true`, still sounds a little bit silly.

After thinking a little bit, I'm not sure we need a comment next to a `true` value. I think it's safe to assume that tests will fail when you change fixtures. I like the comments with valid intervals, because they add extra information. A comment that says "tests will break if you change this" doesn't seem to add information.

**Comment by spark008** [comment]
Ok, so I removed all comments of the sort `# should be ...` from the fixtures. Sorry it took so long to turn this PR around!

**Comment by pwnall** [comment]
Thank you!

**Pull request merged**